components/p3a/histograms_braveizer.cc

```diff
@@ -17,13 +19,13 @@ namespace {
 
 // Please keep this list sorted and synced with |DoHistogramBravezation|.
 // clang-format off
-constexpr const char* kBravezationHistograms[] = {
+constexpr auto kBravezationHistograms = std::to_array<std::string_view>({
```

Comment:
is there a difference from `constexpr std::string_view kBravezationHistograms[] = { ... };`?